tools/macos_retarget_pc_ble_driver_py.sh

```diff
@@ -27,7 +29,7 @@ echo
 
 # Ensure blatann and pc-ble-driver-py are already installed
 BLATANN=`$PYTHON -m pip list | grep blatann`
-PC_BLE_DRIVER_PY=`$PYTHON -m pip list | grep pc-ble-driver-py`
+PC_BLE_DRIVER_PY=`$PYTHON -m pip list | grep pc_ble_driver_py`
```

Comment:
`pip list` for me is showing the name with dashes rather than underscores. What about something like this to support both? 

`grep "\(pc_ble_driver_py\)\|\(pc-ble-driver-py\)"`

Reply:
tested it on mac seems to be working fine :)